app/confirmation/get.njk

```diff
@@ -29,7 +29,7 @@
           </tbody>
         </table>
         <div class="panel panel-border-wide bank-statement-message">
-          <p><span class="bold-small" id="bank-statement-description">'{{description}}'</span> will appear on your bank statement</p>
+          <p>We will notify you by email 3 days before each future payment.</p>
```

Comment:
After clarification, this should be present only for ON_DEMAND. For ONE_OFF nothing should be there

Reply:
Thanks @simad , well spotted 👍